src/oneD/OneDim.cpp

```diff
@@ -220,13 +216,13 @@ int OneDim::solve(double* x, double* xnew, int loglevel)
     return m_newt->solve(x, xnew, *this, *m_jac, loglevel);
 }
 
-void OneDim::evalSSJacobian(double* x, double* xnew)
+void OneDim::evalSSJacobian(double* x, double* r)
```

Comment:
I’m generally not convinced that single-letter variables or functions are a good idea.

Reply:
I renamed it `rsd`, which is what's used a few other places, but just `r` is used in other places in the 1D solver for the residual. Here, `r` is strictly an improvement, as the old name was misleading -- this definitely does not contain a "new" value for `x`.